Testing of ANU-tub using different MOM versions: Jbisits (Issue: #155)

chrisb13 opened the pull request:
Related: #155 

This is for testing.

Note, re: [9385396](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/938539612332491a9ef9159990fc93c2a603efe5):
```yaml
    # used to have: - access-om3@git.2025.08.001
    # Jbisits MOM6 code https://github.com/jbisits/MOM6/commit/3612119ce778e54d0d2fe35011bf42005afd7d0c
    # =2025.07.000 means spack will use that set of dependencies to concretise
    # see https://github.com/ACCESS-NRI/spack-packages/blob/fd44ae03141eab221ce0e5b587dbb9c81d411fcb/packages/access-mom6/package.py#L27
    # link regarding the Debug flag https://spack-tutorial.readthedocs.io/en/latest/tutorial_buildsystems.html#cmake
```

See [here for some more details](https://access-community-hub.github.io/access-iom3-configs/infrastructure/Building/) on how this works.

DO NOT MERGE

---
:rocket: The latest prerelease `access-om3/pr156-6` at a9e7e80a01eda35d6cea9b118aac53539462fd65 is here: https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156#issuecomment-3530340606 :rocket:

chrisb13 committed:
Modify access-mom6 specs and add comments

Updated access-mom6 specifications and added comments for clarity.

chrisb13 commented:
Any thoughts on why this failed @anton-seaice?

Github action code [link](https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19217319406/job/54928851151?pr=156). Commit [9385396](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/938539612332491a9ef9159990fc93c2a603efe5).

```bash
==> [2025-11-10-12:08:27.749113] Error: Terminating after first install failure: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-unroll -mtune=sapphirerapids -march=sapphirerapids' '-DCMAKE_CXX_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/spack-src'
Traceback (most recent call last):
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2118, in install
    self._install_task(task, install_status)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 1672, in _install_task
    pkg, build_process, install_args
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/build_environment.py", line 1374, in start_build_process
    raise child_result
==> [2025-11-10-12:08:05.257038] Updating view at /g/data/vk83/prerelease/apps/spack/0.22/environments/access-om3-pr156-1/.spack-env/view
spack.build_environment.ChildError: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-unroll -mtune=sapphirerapids -march=sapphirerapids' '-DCMAKE_CXX_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/spack-src'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1069, in main
    return _main(argv)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1022, in _main
    return finish_parse_and_run(parser, cmd_name, args, env_format_error)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1052, in finish_parse_and_run
    return _invoke_command(command, parser, args, unknown)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 649, in _invoke_command
    return_val = command(parser, args)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 351, in install
    install_with_active_env(env, args, install_kwargs, reporter_factory)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 420, in install_with_active_env
    env.install_specs(specs_to_install, **install_kwargs)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/environment/environment.py", line 1950, in install_specs
    PackageInstaller([spec.package for spec in specs], install_args).install()
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2169, in install
    raise InstallError(f"{fail_fast_err}: {str(exc)}", pkg=pkg)
spack.installer.InstallError: Terminating after first install failure: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-jyre7v6omd7ra4g42z27j7ozw7uhqcaf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-unroll -mtune=sapphirerapids -march=sapphirerapids' '-DCMAKE_CXX_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-pqgyp7zlgnfpzjhucqqbflqmsachw55j/spack-src'
Error: Process completed with exit code 1.
```

anton-seaice commented:
1 error found in build log:
     6     -- Check for working Fortran compiler: /g/data/vk83/prerelease/apps/
           spack/0.22/spack/lib/spack/env/oneapi/ifx - skipped
     7     CMake Warning at CMakeLists.txt:25 (message):
     8       Compiler not officially supported: IntelLLVM
     9     
     10    
     11    -- Could NOT find MPI_Fortran (missing: MPI_mpi_usempif08_LIBRARY MP
           I_mpi_usempi_ignore_tkr_LIBRARY MPI_mpi_mpifh_LIBRARY MPI_Fortran_WO
           RKS)
  >> 12    CMake Error at /apps/cmake/3.24.2/share/cmake-3.24/Modules/FindPacka
           geHandleStandardArgs.cmake:230 (message):
     13      Could NOT find MPI (missing: MPI_Fortran_FOUND Fortran)
     14    Call Stack (most recent call first):
     15      /apps/cmake/3.24.2/share/cmake-3.24/Modules/FindPackageHandleStand
           ardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
     16      /apps/cmake/3.24.2/share/cmake-3.24/Modules/FindMPI.cmake:1835 (fi
           nd_package_handle_standard_args)
     17      CMakeLists.txt:75 (find_package)
     18    


Appears to be failing [here](https://github.com/ACCESS-NRI/GFDL-generic-tracers/blob/b4cd978e0ff18a34685797bd1a122ee1834a83ab/CMakeLists.txt#L75) in generic tracers

Not sure!

chrisb13 commented:
Thanks @anton-seaice, where and how did you find the build log?

I've also tried with @angus-g's code and it [doesn't work](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/157#issuecomment-3509113942) but it seems to [fail on a different error](https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19217981332/job/54930511533).

So even though we are doing MOM6 standalone generic tracers is included? Isn't that a bit weird?

anton-seaice commented:
> Thanks @anton-seaice, where and how did you find the build log?

Same link - just scroll up further from the bottom e.g. https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19217319406/job/54928851151?pr=156#step:16:1374

There's no option for building without gtracers - sorry I can't remember the history there!

chrisb13 committed:
Update access-mom6 git commit reference in spack.yaml

Just trying something from `main` on the @jbisits fork https://github.com/jbisits/MOM6 (trying to test whether it's just something about not being in our org').

chrisb13 commented:
@chrisb13 to do:

This didn't work.
>[f10352d](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/f10352d31f56769df99abae15747c6df942d395f)
Just trying something from `main` on the @jbisits fork https://github.com/jbisits/MOM6 (trying to test whether it's just something about not being in our org').

Need to find a commit on Joey's repo that is also on ours and so should just work (i.e. test it isn't something weird about being on a fork)

jbisits commented:
> @chrisb13 to do:
> 
> This didn't work.
> 
> > [f10352d](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/f10352d31f56769df99abae15747c6df942d395f)
> > Just trying something from `main` on the @jbisits fork https://github.com/jbisits/MOM6 (trying to test whether it's just something about not being in our org').
> 
> Need to find a commit on Joey's repo that is also on ours and so should just work (i.e. test it isn't something weird about being on a fork)

Let me know if there is anything I can do to help here. Happy to fork ACCESS-MOM6 and move my branch to there if that helps? It might help me because I have been trying to setup an OM3 build today with my fork (also tried with the path to the source on gadi) and was not able to get it setup so far (though that is my issue and not related to this).

chrisb13 commented:
Thanks @jbisits, let me see how I go.

Regarding this point:
> Need to find a commit on Joey's repo that is also on ours

My plan is to use `git merge-base`, [e.g.](https://github.com/ACCESS-NRI/access-om3-configs/issues/880#issuecomment-3507514262):
> git merge-base dev-MC_4km_jra_ryf+regionalpanan dev-MC_4km_jra_ryf+regionalpanan+isf
52bf0b61f8ded1e07dde5646a54cee892a378509

And on:

> I have been trying to setup an OM3 build today with my fork (also tried with the path to the source on gadi) and was not able to get it setup so far (though that is my issue and not related to this).

Okay, well feel free to open an issue (probably in this repo, if it's related to building OM3 like variants), there's lots of Spack related help here.

anton-seaice commented:
> 13 Could NOT find MPI (missing: MPI_Fortran_FOUND Fortran)

I can reproduce this outside spack if i use `CMAKE_Fortran_COMPILER="/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/env/oneapi/ifx"` but it finds OpenMPI fine if i use `CMAKE_Fortran_COMPILER=mpifort`

We don't normally have to specify that anywhere, so its a bit strange

jbisits commented:
@chrisb13 with [`jib/numerical-mixing@6e70504`](https://github.com/jbisits/MOM6/commit/6e705048aa6b400ee7630ef8e2be94d415fff86f) I have fixed the build errors I hit when using spack (see #158). Updating to this commit might help the spack build here.

chrisb13 committed:
Update spack.yaml

Following the @jbisits [suggestion](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156#issuecomment-3516233826). Will try building with the re-based version (https://github.com/jbisits/MOM6/commit/6e705048aa6b400ee7630ef8e2be94d415fff86f)

Related: 
https://github.com/ACCESS-NRI/ACCESS-OM3/issues/158

chrisb13 commented:
> @chrisb13 with [`jib/numerical-mixing@6e70504`](https://github.com/jbisits/MOM6/commit/6e705048aa6b400ee7630ef8e2be94d415fff86f) I have fixed the build errors I hit when using spack (see #158). Updating to this commit might help the spack build here.

H'm [still didn't work unfortunately](https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19283279028/job/55138734640), I get

```bash
==> [2025-11-12-12:28:01.004850] InstallError: Terminating after first install failure: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-mtune=sapphirerapids -march=sapphirerapids -unroll' '-DCMAKE_CXX_FLAGS=-mtune=sapphirerapids -unroll -march=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/spack-src'
==> [2025-11-12-12:28:01.005027] Error: Terminating after first install failure: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-mtune=sapphirerapids -march=sapphirerapids -unroll' '-DCMAKE_CXX_FLAGS=-mtune=sapphirerapids -unroll -march=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/spack-src'
Traceback (most recent call last):
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2118, in install
    self._install_task(task, install_status)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 1672, in _install_task
    pkg, build_process, install_args
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/build_environment.py", line 1374, in start_build_process
    raise child_result
spack.build_environment.ChildError: ProcessError: Command exited with status 1:
==> [2025-11-12-12:27:41.732609] Updating view at /g/data/vk83/prerelease/apps/spack/0.22/environments/access-om3-pr156-3/.spack-env/view
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-mtune=sapphirerapids -march=sapphirerapids -unroll' '-DCMAKE_CXX_FLAGS=-mtune=sapphirerapids -unroll -march=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/spack-src'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1069, in main
    return _main(argv)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1022, in _main
    return finish_parse_and_run(parser, cmd_name, args, env_format_error)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1052, in finish_parse_and_run
    return _invoke_command(command, parser, args, unknown)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 649, in _invoke_command
    return_val = command(parser, args)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 351, in install
    install_with_active_env(env, args, install_kwargs, reporter_factory)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 420, in install_with_active_env
    env.install_specs(specs_to_install, **install_kwargs)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/environment/environment.py", line 1950, in install_specs
    PackageInstaller([spec.package for spec in specs], install_args).install()
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2169, in install
    raise InstallError(f"{fail_fast_err}: {str(exc)}", pkg=pkg)
spack.installer.InstallError: Terminating after first install failure: ProcessError: Command exited with status 1:
    '/apps/cmake/3.24.2/bin/cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a' '-DCMAKE_BUILD_TYPE:STRING=Release' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_POLICY_DEFAULT_CMP0090:STRING=NEW' '-DCMAKE_FIND_USE_PACKAGE_REGISTRY:BOOL=OFF' '-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=ON' '-DCMAKE_INSTALL_RPATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu/lib;/apps/openmpi/4.1.7/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj/lib;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo/lib64;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m/lib64' '-DCMAKE_PREFIX_PATH:STRING=/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/fms-2025.03-tprdjm7hvmienvswdypfmwcnq7twaa4d;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-fortran-4.6.1-kpafmjn6dnwlm4uauyf4iukpskyanlqj;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/netcdf-c-4.9.2-t2t46hwpfaofaffmzgg5invuqhjcqisf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/libaec-1.0.6-jcrrmtkfy4fyr5a44irjev66rbnefx2m;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/hdf5-1.14.3-4zpnxmmmwib3woeyves5g5ujqthekumo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/c-blosc-1.21.5-aumkf5f324nmiisfv72or66nh2ln2kwo;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zstd-1.5.6-u3y3yz5ohcgumfogkiiwye6j2k4qfqo4;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/zlib-ng-2.1.6-nqvthcfjko2yew2ldu27ilkhswxcdogd;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/snappy-1.1.10-qjn5whmsz6aeph6hc4g3n7k3muhuyxqg;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/lz4-1.9.4-zjkyjelpbqblwlgm6xphcp3dj66z6xi2;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/bzip2-1.0.8-tnxlpfbi2vlz2uqmwngr7z5jvetcnl77;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/access-mocsy-2025.07.002-ymzcsnrio3n2tagkvtb6yymabti5b76i;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/gmake-4.4.1-ug7cxkury2dq3xzrvilrezkbojuvai2w;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/oneapi-2025.2.0/intel-oneapi-runtime-2025.2.0-gsm6p2qsbrjruz3rx3juoa6tedqrqrhf;/g/data/vk83/prerelease/apps/spack/0.22/release/linux-rocky8-x86_64_v4/gcc-14.1.0/gcc-runtime-14.1.0-vf3ihhn5dusv3zrxrozoh7cswptn2ptu;/apps/openmpi/4.1.7;/apps/cmake/3.24.2' '-DCMAKE_C_FLAGS=-mtune=sapphirerapids -march=sapphirerapids -unroll' '-DCMAKE_CXX_FLAGS=-mtune=sapphirerapids -unroll -march=sapphirerapids' '-DCMAKE_Fortran_FLAGS=-unroll -march=sapphirerapids -mtune=sapphirerapids' '-DBUILD_SHARED_LIBS:BOOL=OFF' '/scratch/tm70/***/tmp/spack-stage/spack-stage-access-generic-tracers-2025.08.000-m7b7r53a3unultcw2h2k25ma5hoyrq4a/spack-src'
Error: Process completed with exit code 1.
```

@anton-seaice suggested I use `2025.07.000` of the mom6 spack package:
https://github.com/ACCESS-NRI/access-spack-packages/blob/d8b48b18a1400cf2a6850af0bf038af55804d92f/packages/access-mom6/package.py#L27

Do you know what [you used](https://github.com/ACCESS-NRI/ACCESS-OM3/issues/158#issuecomment-3516211095) @jbisits? 

@harshula or @CodeGat will likely have some tips to figure out what is different between your Gadi machine build and my attempt at CI infra here..?

jbisits commented:
I used 2025.07.000 in the `spack.yaml` in the development environment

```terminal
develop:
    access-mom6:
        spec: access-mom6@2025.07.000
        path: /g/data/e14/jb2381/MOM6-examples/src/MOM6-nm
```

jbisits commented:
@chrisb13 I am not sure if this is related to the error here (I think it is not) but thought I would mention it. When I build MOM6 with `ninja`, using mom6-ninja-nci, I cannot have `payu` loaded. This is because it tries to use a different fortran compiler, the one `payu` points to, than the one that is specified by mom6-ninja-nci.

chrisb13 commented:
Thanks for following up @jbisits, an update: @anton-seaice has noticed that we have something wrong with our "online" build in general, both for mom6 standalone and om3, details:
 - https://github.com/ACCESS-NRI/ACCESS-OM3/pull/160
 - https://github.com/ACCESS-NRI/ACCESS-OM3/pull/161
Presumably something changed without us noticing till now. Our Spack wizards are looking into it. My expectation is that once that is fixed we'll be able to get past the above errors but it remains to be seen if it builds without other issues.

chrisb13 commented:
!redeploy

chrisb13 commented:
H'm looks like [the fixes](https://access-nri.zulipchat.com/#narrow/channel/470325-model-release/topic/.60missing.3A.20MPI_Fortran_FOUND.20.60.20in.20spack.20builds/near/555471208) have helped but my impression is that it can't find @jbisits fork:

```python
==> [2025-11-14-10:47:06.998910] Updating view at /g/data/vk83/prerelease/apps/spack/0.22/environments/access-om3-pr156-4/.spack-env/view
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2118, in install
    self._install_task(task, install_status)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 1672, in _install_task
    pkg, build_process, install_args
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/build_environment.py", line 1374, in start_build_process
    raise child_result
spack.build_environment.ChildError: FetchError: All fetchers failed for spack-stage-access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000-y7gmaakv7cph7tsfduswu36c4hc2hgho
        file:///g/data/vk83/prerelease/apps/spack/0.22/buildcache/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz: URLError: <urlopen error [Errno 2] No such file or directory: '/g/data/vk83/prerelease/apps/spack/0.22/buildcache/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz'>
    file:///g/data/vk83/prerelease/apps/spack/0.22/buildcache/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz: URLError: <urlopen error [Errno 2] No such file or directory: '/g/data/vk83/prerelease/apps/spack/0.22/buildcache/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz'>
    https://mirror.spack.io/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz: DetailedHTTPError: GET https://mirror.spack.io/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz returned 404: Not Found
    https://mirror.spack.io/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz: DetailedHTTPError: GET https://mirror.spack.io/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz returned 404: Not Found
    [git] https://github.com/ACCESS-NRI/MOM6.git at commit 6e705048aa6b400ee7630ef8e2be94d415fff86f: ProcessError: Command exited with status 128:
    '/bin/git' '-c' 'advice.detachedHead=false' 'checkout' '6e705048aa6b400ee7630ef8e2be94d415fff86f'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1069, in main
    return _main(argv)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1022, in _main
    return finish_parse_and_run(parser, cmd_name, args, env_format_error)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 1052, in finish_parse_and_run
    return _invoke_command(command, parser, args, unknown)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/main.py", line 649, in _invoke_command
    return_val = command(parser, args)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 351, in install
    install_with_active_env(env, args, install_kwargs, reporter_factory)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/cmd/install.py", line 420, in install_with_active_env
    env.install_specs(specs_to_install, **install_kwargs)
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/environment/environment.py", line 1950, in install_specs
    PackageInstaller([spec.package for spec in specs], install_args).install()
  File "/g/data/vk83/prerelease/apps/spack/0.22/spack/lib/spack/spack/installer.py", line 2169, in install
    raise InstallError(f"{fail_fast_err}: {str(exc)}", pkg=pkg)
spack.installer.InstallError: Terminating after first install failure: FetchError: All fetchers failed for spack-stage-access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000-y7gmaakv7cph7tsfduswu36c4hc2hgho
        file:///g/data/vk83/prerelease/apps/spack/0.22/buildcache/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz: URLError: <urlopen error [Errno 2] No such file or directory: '/g/data/vk83/prerelease/apps/spack/0.22/buildcache/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz'>
    file:///g/data/vk83/prerelease/apps/spack/0.22/buildcache/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz: URLError: <urlopen error [Errno 2] No such file or directory: '/g/data/vk83/prerelease/apps/spack/0.22/buildcache/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz'>
    https://mirror.spack.io/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz: DetailedHTTPError: GET https://mirror.spack.io/_source-cache/git/ACCESS-NRI/MOM6.git/6e705048aa6b400ee7630ef8e2be94d415fff86f.tar.gz returned 404: Not Found
    https://mirror.spack.io/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz: DetailedHTTPError: GET https://mirror.spack.io/access-mom6/access-mom6-git.6e705048aa6b400ee7630ef8e2be94d415fff86f_2025.07.000.tar.gz returned 404: Not Found
    [git] https://github.com/ACCESS-NRI/MOM6.git at commit 6e705048aa6b400ee7630ef8e2be94d415fff86f: ProcessError: Command exited with status 128:
    '/bin/git' '-c' 'advice.detachedHead=false' 'checkout' '6e705048aa6b400ee7630ef8e2be94d415fff86f'
Error: Process completed with exit code 1.
```

Any idea how to fix this @CodeGat or @anton-seaice ? (My memory is that you thought there was a chance this might happen Anton?)

chrisb13 committed:
Update spack.yaml

following @CodeGat's suggestion: https://access-nri.zulipchat.com/#narrow/channel/470325-model-release/topic/.60missing.3A.20MPI_Fortran_FOUND.20.60.20in.20spack.20builds/near/555480162
(I think!)

CodeGat committed:
Update indentation

chrisb13 commented:
Many thanks to @harshula @CodeGat and @anton-seaice for all the help getting this thing (mostly) working!

See related chat:
[#model-release > &#96;missing: MPI_Fortran_FOUND &#96; in spack builds](https://access-nri.zulipchat.com/#narrow/channel/470325-model-release/topic/.60missing.3A.20MPI_Fortran_FOUND.20.60.20in.20spack.20builds/with/555483772)

Currently this is our status (@anton-seaice)
> built a debug build. When we kind of wanted both a release and a debug build, which it hasn't done

(more [tinkering](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/160) needed)

fyi @jbisits

chrisb13 committed:
Update spack.yaml

Turning this off for the time being as it's duplication/unnecessary: https://access-nri.zulipchat.com/#narrow/channel/470325-model-release/topic/.60missing.3A.20MPI_Fortran_FOUND.20.60.20in.20spack.20builds/near/555483354

see also: https://github.com/ACCESS-NRI/ACCESS-OM3/pull/160

chrisb13 commented:
[from](https://access-nri.zulipchat.com/#narrow/channel/470325-model-release/topic/.60missing.3A.20MPI_Fortran_FOUND.20.60.20in.20spack.20builds/near/555484370) @anton-seaice 
>Its worth searching for "concretiz" in the build log from the runner and checking what it sets build_type too now

Okay, so on 
 - the one ([f65a8e6](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/f65a8e630b8a67aa4c8371f8bf2b15d8c7c1abac)) where we had both lines, [I think](https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19349895928/job/55359156920#step:16:238) it's `build_type=Debug`

 - the one ([a9e7e80](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/156/commits/a9e7e80a01eda35d6cea9b118aac53539462fd65)) where we had only one line without the debug, [I think](https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/19350301159/job/55360437178#step:16:812) it's `build_type=Release`

Am I looking in the right places? Just guessing the `deploy to gadi prerelease` is the right bit.

Is that what you were expecting?

Related: https://github.com/ACCESS-NRI/ACCESS-OM3/pull/160

dougiesquire commented:
@chrisb13 can this be closed?

chrisb13 commented:
[Yes](https://github.com/ACCESS-NRI/ACCESS-OM3/pull/218#issuecomment-4203325208), thanks for your patience.
